### ISSUE TITLE
Dyson pure cool platform plugin not showing fan in homekit

### ISSUE THREAD

**Issue by just808b** [hoobs-org/HOOBS#1259]
Hi,

I am trying to setup my dyson fan in HomeKit via hoobs.  I am using the Dyson purecool platform.  I have put in the required details, but nothing shows up.  I have only added a few devices in Hoobs, but they all shown up.  In the log section it stats ‘ [DysonPureCoolPlatform] Got devices from config’. Can’t see any error messages. 

**Expected behavoir**
Fan should be showing up in HomeKit 

**Plugin name**
Dyson purecool platform

**Please report the issue to the plugin developer**
We like to help the community, however some issues are out of our control. Please report this to the plugin developer and include a link here.

**Version**

Hoobs version 3.3.5 

**Did you upgrade**
Please let us know if you upgraded from a previous version.

- [ ] No

**Pervious version**
If you upgraded, please let us know your previous version.

**Did you orginally upgrade to HOOBS 3 from HOOBS 2.1.1?**
Did you perform an upgrade from HOOBS 2.1.1? Some things are different.

- [ ] No

**What device are you using?**
Please let us know the device you are running HOOBS on.
- [ ] HOOBS Box

**Post your config**
This can be found in the interface, Configuration -> Advanced or you can get it via SSH `cat ~/.hoobs/etc/config.json`

* Remove any passwords from the config file. This is public.

```

{
    "platforms": [
        {
            "platform": "DysonPureCoolPlatform",
            "username": "<YOUR-EMAIL-ADDRESS>",
            "password": "<YOUR-PASSWORD>",
            "countryCode": "<COUNTRY-CODE>",
            "devices": [
                {
                    "ipAddress": "XXX.XXX.XXX.XXX",
                    "serialNumber": "XXX-EU-XXXXXXXX",
                    "credentials": null,
                    "enableAutoModeWhenActivating": false,
                    "enableOscillationWhenActivating": false,
                    "isNightModeEnabled": false,
                    "isJetFocusEnabled": false,
                    "isContinuousMonitoringEnabled": false,
                    "isTemperatureSensorEnabled": false,
                    "isHumiditySensorEnabled": false,
                    "isAirQualitySensorEnabled": false,
                    "isSingleAccessoryModeEnabled": false,
                    "isFullRangeHumidity": false,
                    "isHeatingDisabled": false
                }
            ],
            "updateInterval": 60000,
            "retrySignInInterval": 0
        }
    ]
}

```

**Post your log**
You can get the log from the intercafe. This is in the

```

12/05/2021, 17:05:37 [DysonPureCoolPlatform] Checked user account.
12/05/2021, 17:05:37 [DysonPureCoolPlatform] Signing in.
12/05/2021, 17:05:38 [DysonPureCoolPlatform] Signed in.
12/05/2021, 17:05:40 [DysonPureCoolPlatform] No configuration provided for device with serial number C4F-UK-##########
12/05/2021, 17:05:40 [DysonPureCoolPlatform] Got devices from the Dyson API

```

**Additional context**

**Comment by EddieDSuza** [comment]
Try using the [Dyson Pure Cool Plugin](https://plugins.hoobs.org/plugin/homebridge-dyson-pure-cool) as we had some of users with an API error.  

Our recommendation is thru go thru the plugin [issues](https://github.com/lukasroegner/homebridge-dyson-pure-cool/issues) that other users of this plugin are having and if the problem still persists - please open issue with the plugin developer.

**Comment by just808b** [comment]
This is the plug-in I am trying.  I tried restarting the Hoobs device and login and out of the Dyson app and leaving overnight as some people have recommended.  I have changed the password so that no special characters are used.   I have new logs. 

Any help would be appreciated

13/05/2021, 13:54:26 [DysonPureCoolPlatform] Checking user account.
13/05/2021, 13:54:26 Preparing Advertiser for 'HOOBS A389' using bonjour-hap backend!
13/05/2021, 13:54:26 Starting to advertise 'HOOBS A389' using bonjour-hap backend!
13/05/2021, 13:54:26 Bridge is running on port 51826.
13/05/2021, 13:54:27 [TplinkSmarthome] New Device Online: [Intercom] plug [800611A33B42BF3A44C5B6FAA6671C821877A9D7] 192.168.0.19 9999
13/05/2021, 13:54:27 [TplinkSmarthome] Adding: [Intercom] plug [800611A33B42BF3A44C5B6FAA6671C821877A9D7]
13/05/2021, 13:54:27 [DysonPureCoolPlatform] Checked user account.
13/05/2021, 13:54:27 [DysonPureCoolPlatform] Signing in.
13/05/2021, 13:54:27 [DysonPureCoolPlatform] Error while signing in. Status Code: 401
13/05/2021, 13:54:27 [DysonPureCoolPlatform] API could not be reached. Retry is disabled
